app/src/main/AndroidManifest.xml

```diff
@@ -1,21 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="edu.uc.it3048.mypokedex">
-    <uses-permission android:name="android.permission.INTERNET"/>
+<manifest xmlns:android = "http://schemas.android.com/apk/res/android"
+    xmlns:tools = "http://schemas.android.com/tools"
+    package = "edu.uc.it3048.mypokedex">
+    <uses-permission android:name = "android.permission.INTERNET"/>
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
-        <activity android:name=".SavedSightingsActivity" />
-        <activity android:name=".ProfileScreenActivity" />
-        <activity android:name=".MainActivity">
+        android:theme="@style/AppTheme"
+        tools:ignore="AllowBackup">
```

Comment:
This is for the [automatic user data backup](https://developer.android.com/guide/topics/data/autobackup)

Reply:
Hmmm, a good citation.  I was unaware of this feature!